jraft-core/src/main/java/com/alipay/sofa/jraft/core/Replicator.java

```diff
@@ -200,6 +199,82 @@ private ReplicatorMetricSet(final ReplicatorOptions opts, final Replicator r) {
         INSTALLING_SNAPSHOT // installing snapshot
     }
 
+    /**
+     * User can implement the ReplicatorStateListener interface by themselives,
+     * so they can do some their own logic codes when replicator started, had some Error ,stopped.
+     *
+     */
+    public interface ReplicatorStateListener {
+
+        /**
+         * Called when this replicator is started.
+         *
+         */
+        void onStarted();
+
+        /**
+         * Called when this replicator has some errors
+         *
+         * @param status replicator's error detailed status
+         */
+        void onError(final Status status);
+
+        /**
+         * Called when this replicator has stoped
```

Comment:
was stopped ?

Reply:
Okay,I will fix this codes comments and then push it again.Thanks!